### ISSUE TITLE
Why do I get a package named ~QLAlchemy instead of SQLAlchemy

### ISSUE THREAD

**Issue by LianGee** [bndr/pipreqs#206]
```
➜  galio git:(dev) ✗ python3 --version
Python 3.7.1
➜  galio git:(dev) ✗ pipreqs --version                     
0.4.10
```
```
➜  galio git:(dev) ✗ pipreqs ./ --use-local --force --debug
DEBUG: Found packages: {'sqlalchemy', 'traceback', 'jinja2', 'colorama', 'json', 'subprocess', 'mimetypes', 'functools', 'flask_script', 'math', 'flask_cors', 'tarfile', 'shutil', 'pynpm', 'copy', 'time', 'datetime', 'cacheout', 're', 'logging', 'urllib', 'kubernetes', 'mysql', 'requests', 'yaml', 'os', 'git', 'redis', 'flask_socketio', 'socket', 'hashlib', 'apscheduler', 'random', 'flask', 'uuid'}
DEBUG: Found imports: apscheduler, cacheout, colorama, Flask, flask_cors, flask_script, flask_socketio, GitPython, Jinja2, kubernetes, mysql_connector_repackaged, pynpm, PyYAML, redis, Requests, SQLAlchemy
DEBUG: Getting package information ONLY from local installation.
DEBUG: Writing 13 requirements: pynpm, Flask_Cors, Flask_Script, Jinja2, ~QLAlchemy, requests, Flask_SocketIO, kubernetes, redis, cacheout, colorama, APScheduler, Flask to ./requirements.txt
INFO: Successfully saved requirements file in ./requirements.txt
```
 requirement.txt when run the cmd above
```
pynpm==0.1.2
Flask_Cors==3.0.7
Flask_Script==2.0.6
Jinja2==2.10
~QLAlchemy==1.3.15
requests==2.21.0
Flask_SocketIO==4.2.1
kubernetes==11.0.0
redis==3.2.1
cacheout==0.11.1
colorama==0.4.1
APScheduler==3.5.3
Flask==1.0.2
```
why do I get `~QLAlchemy==1.3.15`

**Comment by alan-barzilay** [comment]
Hi! I know this is an old issue but could you provide a minimal setup to reproduce your error?

**Comment by alan-barzilay** [comment]
closing due to inactivity, feel free to reopen if you can provide a minimal setup that can reproduce the error

**Comment by MichaelCurrie** [comment]
I got this persistent warning when running pip in python 3.9:

`WARNING: Ignoring invalid distribution -qlalchemy (c:\programdata\anaconda3\lib\site-packages)`

So I checked and I had a folder called ~qlalchemy-1.4.22.dist-info in my c:\programdata\anaconda3\lib\site-packages.  The tilde was getting reported in the warning as a -.  I'm not sure if this is part of that typosquatting attack mentioned above.  None of the contents of the folder seemed to be dangerous:

![image](https://user-images.githubusercontent.com/5267078/173200170-c693bd5f-7a8a-44e5-99e2-64759a95acc6.png)

The only file contents that might be dangerous was in `direct_url.json` but its entire file contents is just the below, which just pointed to a nonexistent folder on my local drive:

`{"dir_info": {}, "url": "file:///C:/ci/sqlalchemy_1626948551162/work"}`

Anyway I deleted this folder and the warning went away.
